packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/PaymentQueueTests.m

```diff
@@ -61,20 +61,20 @@ - (void)testTransactionPurchased {
         return YES;
       }
       updatedDownloads:nil
-      transactionCache:OCMClassMock(FIATransactionCache.class)];
+      transactionCache:[TestTransactionCache alloc]];
```

Comment:
need `init` here. Can you double check the whole pr and make sure similar issues are addressed.

Reply:
oh oopsies